### ISSUE TITLE
Don't show white background for read-only item pane field

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#4789]
This (from a read-only library) makes it look like the field is editable:

![Screenshot 2024-10-24 at 11 43 00 PM](https://github.com/user-attachments/assets/cc9741df-577b-42e3-ba45-a803ca2c048b)

I think it should be the gray background with the focus ring.
